pkgs/native_assets_cli/example/build/linked_library/README.md

```diff
@@ -0,0 +1,7 @@
+An example library that builds two native libraries where one library
```

Comment:
Should we make the chain 3 long instead of 2, to ensure we support this recursively properly?